platforms/documentation/docs/build.gradle

```diff
@@ -38,7 +38,12 @@ configurations {
     docsTestRuntimeClasspath.extendsFrom(integTestDistributionRuntimeOnly)
 }
 
+configurations.configureEach {
```

Comment:
Why configure all configurations instead of being more targeted?

I think we should be able to exclude from `docsTestImplementation` and that exclusion should be inherited by all configurations that extend from it.